pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1537,6 +1537,12 @@ type ObjectStoreUserSpec struct {
 	// +optional
 	// +nullable
 	Quotas *ObjectUserQuotaSpec `json:"quotas,omitempty"`
+	// +optional
```

Comment:
We also need [Documentation updated](https://github.com/rook/rook/blob/master/Documentation/CRDs/Object-Storage/ceph-object-store-user-crd.md)

Reply:
yes